jekyll/_includes/snippets/runner/install-with-cli-steps.adoc

```diff
@@ -0,0 +1,32 @@
+//== Command line installation: pre-steps
+
+// Display the following step for machine runner installation only as container runner is not yet avaiable on server
+ifdef::machine[]
+NOTE: If you are installing **self-hosted runners for server**, the CircleCI CLI needs to be configured using your server API key. Run `circleci setup` to configure the CLI and access the option to supply a new API token if required.
+endif::[]
+
+In order to install self-hosted runners, you will need to create a namespace and authentication token by performing the steps listed below. Please note that to create resource classes and tokens you need to be an organization administrator in the VCS provider.
+
+You can view your installed runners in the https://app.circleci.com/[web app], by clicking *Self-Hosted Runners* on the left navigation.
```

Comment:
For consistency with the web app instructions: "You can view your runners on the inventory page, by clicking *Self-Hosted Runners* on the left navigation." ?